installer/roles/kubernetes/defaults/main.yml

```diff
@@ -2,6 +2,24 @@
 dockerhub_version: "{{ lookup('file', playbook_dir + '/../VERSION') }}"
 create_preload_data: True
 
+rbac: false
+awx_psp_create: 'false'
```

Comment:
This mixed typing is a little weird.

Reply:
Now that i revisit this, this is alittle weird...
I've changed all instances of string true or false to boolean types